frontend/views/components/GroupSettings/InviteActionButton.vue

```diff
@@ -0,0 +1,60 @@
+<template lang="pug">
+menu-parent
+  menu-trigger.is-icon
+    i.icon-ellipsis-v
+
+  menu-content.c-content
+    ul
+      menu-item(
+        tag='router-link'
+        item-id='hash-3'
+        to='/somewhere'
+      )
+        i18n.icon-vote-yea.c-icon(tag='i') See original proposal
+      menu-item(
+        v-if='!linkExpired'
+        tag='router-link'
+        item-id='hash-3'
+        to='/somewhere'
+      )
+        i18n.icon-times.c-icon(tag='i') Revoke Link
+</template>
+
+<script>
+import { MenuParent, MenuTrigger, MenuContent, MenuItem } from '@components/Menu/index.js'
+
+export default {
+  name: 'InviteLinksActionButton',
+  components: {
+    MenuParent,
+    MenuTrigger,
+    MenuContent,
+    MenuItem
+  },
+  props: {
+    linkExpired: Boolean
```

Comment:
it's a common convention among developers to have Booleans starting with `is` or `has` so you easily understand it's a boolean. in this case `isLinkExpired`

Reply:
Fixed! :)